docs/getting-started.md

```diff
@@ -143,11 +143,10 @@ process via HTTP requests:
 API_SOCKET="/tmp/firecracker.socket"
 
 # Remove API unix socket
-rm -f $API_SOCKET
+sudo rm -f $API_SOCKET
```

Comment:
would it be a bad idea to remove all sudoes and let users manage their own permissions? Now it looks like it's mandatory, which may scare some of the users. We can leave a note saying something like "depending on your permissions management, you may need to run some of these with superuser rights". I'm not fully convinced either way at the moment.

Reply:
the typical approach for the getting started page is to run the firecracker binary without the jailer and so we need to add sudo to run the commands. I usually copy paste the command to a notepad and add the sudo to it to run. This change would save that time.
And we already ask them to run a bunch of commands with sudo later in the doc.